SDKCLI/Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "SDKCLI",
```

Comment:
as from start we must call AWSSDKCLI

all caps looks terrible through

we can imagine SDKCLI in smithy-swift